CHANGELOG.md

```diff
@@ -1,5 +1,42 @@
 # Changelog
 
+## [1.49](https://github.com/brave/brave-ios/milestone/106)
+
+Web3
+
+- Added Solana Swap integration into existing Swap UI using Jupiter as the DEX aggregator. ([#6847](https://github.com/brave/brave-ios/issues/6847))
+- Added transaction submitted and transaction completed state in wallet panel. ([#6881](https://github.com/brave/brave-ios/issues/6861))
+- Added UI to display "signed" status for Ethereum transactions. ([#6606](https://github.com/brave/brave-ios/issues/6606))
+- Added support for sending to ENS wallet address. ([#5787](https://github.com/brave/brave-ios/issues/5787))
+- Added auto-complete when entering token address for custom Solana assets. ([#6982](https://github.com/brave/brave-ios/issues/6982))
+- Added support for Solana Name Service (SNS) navigation in app. ([#6706](https://github.com/brave/brave-ios/issues/6706))
+- Fixed wallet issue where the Origin Password field was shown when adding Solana Account. ([#7175](https://github.com/brave/brave-ios/issues/7175))
+- Fixed reporting Brave wallet usage stats. ([#7143](https://github.com/brave/brave-ios/issues/7143))
+- Fixed issue adding a custom network via panel. ([#6965](https://github.com/brave/brave-ios/issues/6965))
+- Fixed issue with ERC721 NFTs with different token identifiers showing incorrect balance. ([#6954](https://github.com/brave/brave-ios/issues/6954))
+- Fixed crash related to loading URLs when selecting one of the on-ramp purchase options. ([#6993](https://github.com/brave/brave-ios/issues/6993))
+
+General UI
```

Comment:
Lets change this to `General` as there's other issues in this category other than just UI. For example, dropping `IOS 14` support and a few others. Also, lets get these bolded like we do on `Desktop` & `Android` using `###`.